internal/kubelet/image-credential-provider.go

```diff
@@ -83,7 +86,10 @@ func generateImageCredentialProviderConfig(cfg *api.NodeConfig, ecrCredentialPro
 	var imageCredentialProviderTemplate *template.Template
 	if cfg.IsIAMRolesAnywhere() {
 		templateVars.AwsConfigPath = cfg.Spec.Hybrid.IAMRolesAnywhere.AwsConfigPath
+		templateVars.AwsProfile = iamrolesanywhere.ProfileName
+		templateVars.AwsCredentialsPath = iamrolesanywhere.EksHybridAwsCredentialsPath
```

Comment:
any way we can inject these in the constructor? just so we don't make the kuebelet package depend on iamrolesanywhere